compose.go

```diff
@@ -89,6 +105,46 @@ func (dc *LocalDockerCompose) getDockerComposeEnvironment() map[string]string {
 	return environment
 }
 
+func (dc *LocalDockerCompose) applyStrategyToRunningContainer() error {
+
+	cli, err := client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		return err
+	}
+
+	for k := range dc.WaitStrategyMap {
+		// Docker compose appends "_1" to every started service by default. Trimming to match running docker container
+		f := filters.NewArgs(filters.Arg("name", strings.TrimSuffix(k.service, "_1")),
```

Comment:
What do you think about hiding the `_1` from the service name instead, so that consumers don't have to remember about that, just passing service name?

If so, we should update the tests in consequence.